templates/shared-infra/enmasse.io_messagingaddresses.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
@lulf this will be neverending battle... someone will generate crds with version 0.3.0 and someone with 0.2.5

Reply:
Sorry, I will fix my local setup :)

Reply:
Fixed